DemoServer/AdditionalLanguages/nodejs/demo/queries/queryExample/metadata.json

```diff
@@ -0,0 +1,151 @@
+{
+    "Slug": "query-example",
+    "SourceFileName": "index.js",
+    "Title": "Query Example",
+    "DescriptionLines": [
+        "This demo is an example of a<br>",
+        " **Query with Combined Features**.",
```

Comment:
How about:
```suggestion
        " **Query with all features combined**.",
```

Reply:
imho - I would prefer Not to use '**all**' - since there are more options that are not demonstrated in the demo, i.e. skip and some others.

maybe we should switch to:
**Query with multiple conditions/options**  :)

Reply:
ok